io.gitlab.o20.word.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "io.gitlab.o20.word",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "word64.run",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--device=dri", "--filesystem=host"],
```

Comment:
Any reason for requiring host access? This went unanswered in the first review.

Reply:
Also, mind putting each permission per line? :)

Reply:
No, I don't think I need host access. I just though it was why something was happening when I was fixing the seg faults, but now I see I was wrong.